pkg/plugin/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,210 @@
+package bootstrap
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"sort"
+	"text/template"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/docker/infrakit/pkg/plugin/group/types"
+	"github.com/docker/infrakit/pkg/spi/group"
+	spi_resource "github.com/docker/infrakit/pkg/spi/resource"
```

Comment:
FWIW, Go convention is to avoid `snake_case`, this would usually just be `spiresource` (standard lib has `httptest` for instance)